azurerm/internal/services/cognitive/cognitive_account_resource.go

```diff
@@ -146,7 +227,46 @@ func resourceCognitiveAccount() *pluginsdk.Resource {
 				},
 			},
 
-			"custom_subdomain_name": {
+			"public_network_access_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+
+			"restrict_outbound_network_access": {
```

Comment:
could we put restrict on the end to be consistant with other bools?
```suggestion
			"outbound_network_access_restrited": {
```